x-pack/plugin/esql/qa/testFixtures/src/main/resources/chunk.csv-spec

```diff
@@ -2,12 +2,12 @@
 # Tests for Chunk function
 #
 
-chunkExample
+chunkDefaults
 required_capability: chunk_function
```

Comment:
the bwc tests fail when running in a mixed cluster mode.
the way we mitigate this in ES|QL is by using capabilities.
when it's an already released feature (not only available in snapshot), we add a new capability that we then refer to it in the tests.

when a feature is in snapshot, we usually don't add a new capability, but we "bump" the existing capability e.g. `chunk_function_v2`.
we need to do the same here - change the CHUNK_FUNCTION capability to CHUNK_FUNCTION_V2 and then rename it in the tests.